doc/source/tutorial/optimize.rst

```diff
@@ -1405,13 +1408,15 @@ should be
         c = [-29, -45, 0, 0]
 
 Next, let's consider the two inequality constraints. These can be converted by using the "greater than" inequality
-constraint to a "less than" inequality constraint by multiplying both sides by a factor of :math:`-1`:
+constraint to a "less than" inequality constraint by multiplying both sides by a factor of :math:`-1`. The first
+one is the "less than" inequality constraint, however second one is the "greater than" inequality constraint. So, only
+second one should be converted to align inequality sign direction:
```

Comment:
Next, let's consider the two inequality constraints. The first one is a "less than" inequality; however, the second one is a "greater than" inequality. `linprog` only accepts "less than" inequality constraints, so we multiply both sides of the second inequality by :math:`-1` to get:

Reply:
I fixed.